docs/reference/components/transforms/reduce.cue

```diff
@@ -37,10 +37,22 @@ components: transforms: reduce: {
 	configuration: {
 		ends_when: {
 			common:      false
-			description: "A condition used to distinguish the final event of a transaction. If this condition resolves to `true` for an event, the current transaction is immediately flushed with this event."
+			description: """
+				A condition used to distinguish the final event of a transaction. If this condition resolves to `true`
+				for an event, the current transaction is immediately flushed with this event.
+
+				This field accepts a [Vector Remap Language](\(urls.vrl_reference)) (VRL) _boolean expression_.
+				Please refer to the [Vector Remap Language reference](\(urls.vrl_reference)) for a list of
+				expressions and functions.
```

Comment:
Can this fragment be easily extracted for re-use?

Reply:
It can! I'm going to add a `remap` boolean field that will accomplish that.